apps/vr-tests-react-components/src/stories/Badge/CounterBadge.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
Should this file go under a `CounterBadge` folder instead of under the `Badge` one?

Also, independent of that, shouldn't the `src/stories/CounterBadge.stories.tsx` have been deleted by the script?

Reply:
Good call on both! I've moved the `CounterBadge` stories in its own subfolder and deleted `stories/CounterBadge.stories.tsx`